route/include/route_generator_worker.h

```diff
@@ -153,6 +155,21 @@ namespace route {
          */
         visualization_msgs::MarkerArray compose_route_marker_msg(const lanelet::Optional<lanelet::routing::Route>& route);
 
+        /**
+        * \brief crosstrack_error_check is a function that determines when the vehicle has left the route and reports when a crosstrack error has
+        * taken place
+        * 
+        *  \param msg Msg that contains the vehicle's current position
+        *  \param current_llt The lanelet that the vehicle is currently in
+        *  \param llt_track The crosstrack and downtrack distance of the current lanelet
+        * */
+        bool crosstrack_error_check(const geometry_msgs::PoseStampedConstPtr& msg, lanelet::ConstLanelet current_llt, carma_wm::TrackPos llt_track);
+
+        void setCTEcounter(int cte_counter);
+
+        void set_out_counter(int out_counter);
```

Comment:
Formatting consistencies between setCTEcounter and set_out_counter, also need doc comments.